### PR TITLE
Developers pages

### PR DESCRIPTION
**Pull request by apt105**
Moved some docs from "other" to the developers section, some new pages on building development environment, and some small fixes to the antenna testing page and the RAK board page. https://meshtastic-git-developers-apt105.vercel.app/

### CODE REVIEW

**Comment by sachaw** [comment]
Let me know when you're ready to merge

File: website/docs/developers/android/build.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+* (unfortunately) you need to get a (free) mapbox developer token [here](https://docs.mapbox.com/android/maps/guides/install/) and put that token in your user gradle.properties.
+```
+~/development/meshtastic/MeshUtil$ cat ~/.gradle/gradle.properties
+MAPBOX_DOWNLOADS_TOKEN=sk.yourtokenherexxx
+```
+
+* Now you should be able to select "Run / Run" in the IDE and it will happily start running on your phone
+or the emulator.  Note: The emulators don't support bluetooth, so some features can not be used in
+that environment.
```
Comment (jfirwin) [inline website/docs/developers/android/build.md:35]: should we break these line breaks?

Comment (jfirwin) [reply website/docs/developers/android/build.md:35]: actually, the note could probably be added to an admonition

File: website/docs/hardware/antenna/testing.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ On the topic of testing - performing your own testing and providing feedback is
 
 ## Signal strength testing
 
-Real world testing is also discussed by Andreas Speiss (the 'guy with the Swiss accent') in his [tutorial](https://www.youtube.com/watch?v=J3PBL9oLPX8). He has written [code](https://github.com/SensorsIot/Antenna-Tester) for testing antennas using two Lora32 V1 boards to compare how different antennas behave.
+Real world testing is also discussed by Andreas Speiss (the 'guy with the Swiss accent') in his [tutorial](https://www.youtube.com/watch?v=J3PBL9oLPX8). He has written [code](https://github.com/SensorsIot/Antenna-Tester) for testing antennas using two Lora32 V1 boards to compare how different antennas behave. Lilygo have also made code available for testing the rssi on the [LORA32](https://github.com/LilyGO/TTGO-LORA32) [boards](https://github.com/Xinyuan-LilyGO/TTGO-LoRa-Series) and the [T-Beam](https://github.com/LilyGO/TTGO-T-Beam).
```
Comment (jfirwin) [inline website/docs/hardware/antenna/testing.md:25]: I'd forgotten all about this, I'm going to need to check it out!

**Review by jfirwin — COMMENTED** [review]
super minor notes. Everything looks really good!

Those dang links never behave how I expect them either haha

File: website/docs/developers/android/build.md (hunk 1 above)
Comment (apt105) [reply website/docs/developers/android/build.md]: Good point, done

File: website/docs/hardware/antenna/testing.md (hunk 2 above)
Comment (apt105) [reply website/docs/hardware/antenna/testing.md:25]: I got a pair of TTGO Lora v1.0 boards just for this purpose! Works fairly well, just need to set up a permanent testing rig.

**Comment by apt105** [comment]
Fixes done, I think it's ready to merge now.

**Pull request merged**